roofit/roofitcore/test/testNaNPacker.cxx

```diff
@@ -101,6 +101,8 @@ TEST(RooNaNPacker, CanPackStuffIntoNaNs)
       dumpFloats(rnp._payload);
 }
 
+#if !defined(_MSC_VER) || defined(R__ENABLE_BROKEN_WIN_TESTS)
```

Comment:
```suggestion
#if !defined(_MSC_VER) || defined(_WIN64) || defined(NDEBUG) || defined(R__ENABLE_BROKEN_WIN_TESTS)
```
if it's failing only with Windows 32 bit Debug builds

Reply:
Thanks for the suggestion!